examples/dump_channel.rs

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+
+use futures::stream::TryStreamExt;
+
+// Once we find a way to load netsimdev kernel module in CI, we can convert this
```

Comment:
s/netsimdev/netdevsim/

Reply:
This comment is copied from other examples provided in the project.
I'll submit a dedicated PR to fix this in all the code examples.